app/assets/stylesheets/container_topology.css

```diff
@@ -19,16 +20,33 @@ kubernetes-topology-graph {
 
 .checkbox-inline {
     font-size: 16px;
-    padding-left: 32px;
+    padding-right: 10px;
 }
 
-.refresh {
+.fa-refresh {
```

Comment:
Well, even if that were the case, it would still make it impossible to use that icon anywhere else in topology. 

But I don't think you can rely on that, there are still common parts of the UI which are visible even in topology, and if you ever do an AJAX transition anywhere else, you'll still have the stylesheet loaded. I'd say in general, it's safer to assume all CSS is global.